### PR TITLE
dev-lang/mono: Multilib support

### PR DESCRIPTION
**Pull request by msva**

### CODE REVIEW

**Comment by monsieurp** [comment]
Thanks. Could you please file a bug as well? None of the maintainers of the Dotnet project has a Github account.

File: dev-lang/mono/mono-4.4.1.0.ebuild [hunk 1]
```diff
@@ -3,10 +3,8 @@
 # $Id$
 
 EAPI=6
-AUTOTOOLS_PRUNE_LIBTOOL_FILES="all"
-AUTOTOOLS_AUTORECONF=1
```
Comment (mgorny) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: Could you split that into a separate commit? It's not really related to multilib but a generic cleanup.

File: dev-lang/mono/mono-4.4.1.0.ebuild [hunk 2]
```diff
@@ -50,6 +45,11 @@ pkg_setup() {
 	mono-env_pkg_setup
 }
 
+src_unpack() {
+	default
+	mv "${WORKDIR}/${PN}-$(get_version_component_range 1-3)" "${S}"
```
Comment (mgorny) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: Why are you changing this?

File: dev-lang/mono/mono-4.4.1.0.ebuild [hunk 3]
```diff
@@ -62,32 +62,14 @@ src_prepare() {
...
-
-	# Fix VB targets
-	# http://osdir.com/ml/general/2015-05/msg20808.html
-	#eapply "${FILESDIR}/add_missing_vb_portable_targets.patch"
-
-	# Fix build when sgen disabled
-	# https://bugzilla.xamarin.com/show_bug.cgi?id=32015
-	#eapply "${FILESDIR}/${PN}-4.0.2.5-fix-mono-dis-makefile-am-when-without-sgen.patch"
-
-	# TODO: update patch
```
Comment (mgorny) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: You seem to be removing TODOs. Not sure if maintainers will like that.

**Comment by msva** [comment]
@monsieurp Actually, I do. And I pointed this PR there. https://bugs.gentoo.org/show_bug.cgi?id=589342

File: dev-lang/mono/mono-4.4.1.0.ebuild (hunk 1 above)
Comment (msva) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: okay

File: dev-lang/mono/mono-4.4.1.0.ebuild (hunk 2 above)
Comment (msva) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: in my expirience, changing default S adds additional troubles with multilib. But if this change is unlikely, I can revert it...

File: dev-lang/mono/mono-4.4.1.0.ebuild (hunk 1 above)
Comment (msva) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: done

**Comment by mpkh** [comment]
@monsieurp actually dotnet dev has github account and also there is dotnet overlay in this organization, I suggest first PR there. sadly dotnet team is really small.

**Comment by mpkh** [comment]
merged, thanks!
